recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+gnuradio-core:
```

Comment:
Needs to use an underscore (for some YAML reason): `gnuradio_core`. Then you should see that it worked after a rerender.

Reply:
thanks :)